src/diffusers/schedulers/scheduling_lms_discrete.py

```diff
@@ -124,10 +124,7 @@ def step(
         return {"prev_sample": prev_sample}
 
     def add_noise(self, original_samples, noise, timesteps):
-        alpha_prod = self.alphas_cumprod[timesteps]
-        alpha_prod = self.match_shape(alpha_prod, original_samples)
-
-        noisy_samples = (alpha_prod**0.5) * original_samples + ((1 - alpha_prod) ** 0.5) * noise
+        noisy_samples = original_samples + noise * self.sigmas[timesteps]
```

Comment:
@nicolas-dufour could you add a quick snippet of how you use the `add_noise` function in your experiments? Because at the moment it fails for batched inputs, and I would expect it to be implemented like so: 

```suggestion
        sigmas = self.match_shape(self.sigmas, noise)
        noisy_samples = original_samples + noise * sigmas[timesteps]
```

Reply:
Hey, I'm doing the following:
I have ```init_latents``` of batch_size 2. and then
```python
init_timestep = int(num_inference_steps * strength)+ offset
init_timestep = min(init_timestep, num_inference_steps)
timesteps = self.scheduler.timesteps[-init_timestep]
t_start = max(num_inference_steps - init_timestep + offset, 0)

timesteps = torch.tensor([timesteps] * batch_size, dtype=torch.long, device=self.device)
noise = torch.randn(init_latents.shape, generator=generator, device=self.device)
init_latents = init_latents+noise* self.scheduler.sigmas[t_start]
```